frontend/src/scenes/experiments/experimentLogic.tsx

```diff
@@ -617,6 +618,42 @@ export const experimentLogic = kea<experimentLogicType>([
                 return !!experiment?.start_date
             },
         ],
+        statusTag: [
+            (s) => [s.experiment, s.isExperimentRunning],
+            (experiment, isExperimentRunning): ReactElement => {
+                const statusColors = { running: 'green', draft: 'default', complete: 'purple' }
+                const status = (): string => {
+                    if (!isExperimentRunning) {
+                        return 'draft'
+                    } else if (!experiment?.end_date) {
+                        return 'running'
+                    }
+                    return 'complete'
+                }
+
+                return (
+                    <Tag style={{ alignSelf: 'center' }} color={statusColors[status()]}>
+                        <b className="uppercase">{status()}</b>
+                    </Tag>
+                )
+            },
+        ],
+        resultsTag: [
+            (s) => [s.experiment, s.experimentResults, s.areResultsSignificant],
+            (experiment, experimentResults, areResultsSignificant): ReactElement => {
+                if (experimentResults && experiment.end_date) {
+                    return (
+                        <Tag style={{ alignSelf: 'center' }} color={areResultsSignificant ? 'green' : 'geekblue'}>
```

Comment:
I think it would be more conventional to move this to the `NotebookNodeExperiment` component:

```
const logic = experimentLogic({ experimentId: id })
const { experiment, experimentResults, areResultsSignificant } = useValues(logic)

...

{experimentResults && experiment.end_date ? (
  <Tag style={{ alignSelf: 'center' }} color={areResultsSignificant ? 'green' : 'geekblue'}>
      <b className="uppercase">
          {areResultsSignificant ? 'Significant Results' : 'Results not significant'}
      </b>
  </Tag>
) : null}
```

Reply:
makes sense 😅